### ISSUE TITLE
Error when specifying multiple services in IngressRoute

### ISSUE THREAD

**Issue by vikas027** [containous/traefik#5803]
### Do you want to request a *feature* or report a *bug*?
Bug

### What did you do?
Error when specifying multiple services in IngressRoute

### What did you expect to see?
Traefik loadbalancing between two services 

### What did you see instead?
```
time="2019-11-07T05:48:52Z" level=error msg="Error in Go routine: runtime error: invalid memory address or nil pointer dereference"
time="2019-11-07T05:48:52Z" level=error msg="Stack: goroutine 53 [running]:\nruntime/debug.Stack(0xc0000fc000, 0x24d0e85, 0x17)\n\t/usr/local/go/src/runtime/debug/stack.go:24 +0x9d\ngithub.com/containous/traefik/v2/pkg/safe.defaultRecoverGoroutine(0x209b8c0, 0x3ee5610)\n\t/go/src/github.com/containous/traefik/pkg/safe/routine.go:160 +0xb1\ngithub.com/containous/traefik/v2/pkg/safe.GoWithRecover.func1.1(0x25ab3a0)\n\t/go/src/github.com/containous/traefik/pkg/safe/routine.go:150 +0x57\npanic(0x209b8c0, 0x3ee5610)\n\t/usr/local/go/src/runtime/panic.go:679 +0x1b2\ngithub.com/containous/traefik/v2/pkg/metrics.OnConfigurationUpdate(0xc00088a3f0, 0xc00055b400, 0x5, 0x8)\n\t/go/src/github.com/containous/traefik/pkg/metrics/prometheus.go:232 +0x618\ngithub.com/containous/traefik/v2/pkg/server.(*Server).loadConfiguration(0xc0003b8000, 0x24bb87e, 0xd, 0xc000810560)\n\t/go/src/github.com/containous/traefik/pkg/server/server_configuration.go:59 +0x49a\ngithub.com/containous/traefik/v2/pkg/server.(*Server).listenConfigurations(0xc0003b8000, 0xc00047b5e0)\n\t/go/src/github.com/containous/traefik/pkg/server/server_configuration.go:249 +0x51\ngithub.com/containous/traefik/v2/pkg/server.(*Server).Start.func3(0xc00047b5e0)\n\t/go/src/github.com/containous/traefik/pkg/server/server.go:183 +0x34\ngithub.com/containous/traefik/v2/pkg/safe.(*Pool).Go.func1()\n\t/go/src/github.com/containous/traefik/pkg/safe/routine.go:90 +0x7b\ngithub.com/containous/traefik/v2/pkg/safe.GoWithRecover.func1(0x25ab3a0, 0xc00012b8c0)\n\t/go/src/github.com/containous/traefik/pkg/safe/routine.go:153 +0x57\ncreated by github.com/containous/traefik/v2/pkg/safe.GoWithRecover\n\t/go/src/github.com/containous/traefik/pkg/safe/routine.go:147 +0x49\n"
```

```
runtime/debug.Stack(0xc0000fc000, 0x24d0e85, 0x17)
	/usr/local/go/src/runtime/debug/stack.go:24 +0x9d
github.com/containous/traefik/v2/pkg/safe.defaultRecoverGoroutine(0x209b8c0, 0x3ee5610)
	/go/src/github.com/containous/traefik/pkg/safe/routine.go:160 +0xb1
github.com/containous/traefik/v2/pkg/safe.GoWithRecover.func1.1(0x25ab3a0)
	/go/src/github.com/containous/traefik/pkg/safe/routine.go:150 +0x57
panic(0x209b8c0, 0x3ee5610)
	/usr/local/go/src/runtime/panic.go:679 +0x1b2
github.com/containous/traefik/v2/pkg/metrics.OnConfigurationUpdate(0xc00088a3f0, 0xc00055b400, 0x5, 0x8)
	/go/src/github.com/containous/traefik/pkg/metrics/prometheus.go:232 +0x618
github.com/containous/traefik/v2/pkg/server.(*Server).loadConfiguration(0xc0003b8000, 0x24bb87e, 0xd, 0xc000810560)
	/go/src/github.com/containous/traefik/pkg/server/server_configuration.go:59 +0x49a
github.com/containous/traefik/v2/pkg/server.(*Server).listenConfigurations(0xc0003b8000, 0xc00047b5e0)
	/go/src/github.com/containous/traefik/pkg/server/server_configuration.go:249 +0x51
github.com/containous/traefik/v2/pkg/server.(*Server).Start.func3(0xc00047b5e0)
	/go/src/github.com/containous/traefik/pkg/server/server.go:183 +0x34
github.com/containous/traefik/v2/pkg/safe.(*Pool).Go.func1()
	/go/src/github.com/containous/traefik/pkg/safe/routine.go:90 +0x7b
github.com/containous/traefik/v2/pkg/safe.GoWithRecover.func1(0x25ab3a0, 0xc00012b8c0)
	/go/src/github.com/containous/traefik/pkg/safe/routine.go:153 +0x57
created by github.com/containous/traefik/v2/pkg/safe.GoWithRecover
	/go/src/github.com/containous/traefik/pkg/safe/routine.go:147 +0x49
```

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
v2.0.4
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

_My Services_
```bash
$ kubectl get svc -l app=test
NAME       TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
vikas-v1   ClusterIP   10.43.82.235    <none>        80/TCP    57s
vikas-v2   ClusterIP   10.43.113.218   <none>        80/TCP    57s
$
```

_One service works fine_

```yaml
---
apiVersion: traefik.containo.us/v1alpha1
kind: IngressRoute
metadata:
  name: app-test
  namespace: default
spec:
  entryPoints:
  - websecure
  tls: {}
  routes:
    - kind: Rule
      match: Host(`app-test.xx.yy`)
      services:
      - name: vikas-v1
        port: 80
```

_Two services throws an exception_

```yaml
---
apiVersion: traefik.containo.us/v1alpha1
kind: IngressRoute
metadata:
  name: app-test
  namespace: default
spec:
  entryPoints:
  - websecure
  tls: {}
  routes:
    - kind: Rule
      match: Host(`app-test.xx.yy`)
      services:
      - name: vikas-v1
        port: 80
      - name: vikas-v2
        port: 80
```

Same issue is also posted [here](https://community.containo.us/t/error-when-specifying-multiple-services-in-ingressroute/2837).

**Comment by ldez** [comment]
Hello,
Could you provide the full log and stack trace ?

**Comment by vikas027** [comment]
Hey @ldez ,

Sure, I can do that. Can you please let me know how to do the same? :)

**Comment by Ullaakut** [comment]
Hi @vikas027 !

You just need to provide the same logs as you did when opening the issue, in the `What did you see instead?` code block where you pasted two lines of log, except without truncating them to keep only two lines.

This time, please paste all of the lines of logs, from the start of traefik to the moment you encounter the error :)

**Comment by ldez** [comment]
Duplicate of #5755, #5755

Closed by #5759

**Comment by vikas027** [comment]
Thanks, @ldez !!
